Submitted By: Mario Fetka (mario-fetka at gmx dot at)
Date: 2008-07-18
Initial Package Version: 5.11.08
Origin: me
Upstream Status: unknown
Description: add back remotesand correct arm dir
 
diff -Naur makefile.orig /makefile
--- makefile.orig	2007-10-29 16:38:11.000000000 +0000
+++ makefile	2008-07-18 18:42:52.000000000 +0000
@@ -8,7 +8,7 @@
 LIBS =
 ODIR = x86
 ODIR64 = x64
-ODIRARM = arm
+ODIRARM = n800
 
 _OBJS = fileio.o linuxserio.o lowlevel.o server.o errormessage.o flashrom.o webserver.o xap.o lanio.o ascii.o
 OBJS = $(patsubst %,$(ODIR)/%,$(_OBJS))
diff -Naur remotes/ccf.rem.orig remotes/ccf.rem
--- remotes/ccf.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/ccf.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,24 @@
+[REMOTE]
+  [NAME]ccf
+
+
+[COMMANDS]
+  [v+][CCF]0000 006d 0022 0002 0155 00aa 0016 003f 0016 0015 0016 003f 0016 003f 0016 003f 0016 003f 0016 003f 0016 0015 0016 0015 0016 003f 0016 0015 0016 0015 0016 0015 0016 0015 0016 0015 0016 003f 0016 003f 0016 0015 0016 003f 0016 003f 0016 0014 0016 0014 0016 0014 0016 003f 0016 0015 0016 003f 0016 0015 0016 0015 0016 003f 0016 003f 0016 003f 0016 0015 0016 05e7 0155 0055 0016 0e3b
+  [31.63][CCF]5000 0000 0000 0001 001f 003f
+  [0.0][CCF]5000 0000 0000 0001 0000 0000
+  [1.1][CCF]5000 0000 0000 0001 0001 0001
+  [2.2][CCF]5000 0000 0000 0001 0002 0002
+  [31.127][CCF]5000 0000 0000 0001 001f 007f
+  [p+][CCF]5000 0000 0000 0001 0005 0020
+  [p-][CCF]5000 0000 0000 0001 0005 0021
+  [63.63][CCF]6000 0000 0000 0001 003f 003f
+  [255.255][CCF]6000 0000 0000 0001 00ff 00ff
+  [3.3][CCF]6000 0000 0000 0001 0003 0003
+  [vol+][CCF]0000 0067 0000 000d 0060 0018 0018 0018 0030 0018 0018 0018 0018 0018 0030 0018 0018 0018 0018 0018 0030 0018 0018 0018 0018 0018 0018 0018 0018 041e
+  [vol-][CCF]0000 0067 0000 000d 0060 0018 0030 0018 0030 0018 0018 0018 0018 0018 0030 0018 0018 0018 0018 0018 0030 0018 0018 0018 0018 0018 0018 0018 0018 0406
+  [1][CCF]0100 0008 0004 0000 0008 0214 0008 0214 0008 0214 0008 0214
+  [2][CCF]0100 0008 0004 0000 0008 0102 0008 0102 0008 0102 0008 0102
+  [3][CCF]0100 0008 0004 0000 0008 007e 0008 007e 0008 007e 0008 007e
+  [4][CCF]0100 0008 0004 0000 0008 003c 0008 003c 0008 003c 0008 003c
+  [T1][CCF]0000 007D 00A4 0000 00A7 0048 000C 003C 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000D 03D4 00A7 0048 000C 003C 000C 0018 000D 0017 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000D 0017 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000D 003B 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 0018 000D 0017 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000D 0017 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000D 0017 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000D 0017 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 0018 000C 0000 
+  [T2][CCF]0000 007D 00A4 0000 00A7 0048 000C 003C 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000D 03D4 00A7 0048 000C 003C 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000C 003C 000C 0018 000C 003C 000C 003C 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 003C 000C 0018 000C 003C 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 0018 000C 0018 000C 0018 000C 003C 000C 0018 000C 0018 000C 003C 000C 003C 000C 003C 000C 0018 000C 0000 
\ No newline at end of file
diff -Naur remotes/dbox2.rem.orig remotes/dbox2.rem
--- remotes/dbox2.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/dbox2.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,33 @@
+[REMOTE]
+  [NAME]dbox2
+
+[TIMING]
+  [0][N]7[1]504 2488[2]496 504[3]504 992[4]1008 496[5]1000 992[6]504 20424[7]504 79760[RC]1[RP]77[FREQ]38[SB]
+
+[COMMANDS]
+  [1][T]0[D]S1200000000000004S010000000020135S1200000000000000
+  [2][T]0[D]S1200000000000004S13000000020135S1200000000000000
+  [3][T]0[D]S1200000000000004S001000000020135S1200000000000000
+  [4][T]0[D]S1200000000000004S10300000020135S1200000000000000
+  [5][T]0[D]S1200000000000004S01300000020135S1200000000000000
+  [7][T]0[D]S1200000000000004S000100000020135S1200000000000000
+  [9][T]0[D]S1200000000000004S01030000020135S1200000000000000
+  [0][T]0[D]S1200000000000004S100000000020135S1200000000000000
+  [home][T]0[D]S1200000000000004S10000300020135S1200000000000000
+  [dbox][T]0[D]S1200000000000004S00010300020135S1200000000000000
+  [6][T]0[D]S1200000000000004S12100000020135S1200000000000000
+  [8][T]0[D]S1200000000000004S10030000020135S1200000000000000
+  [power][T]0[D]S1200000000000004S10210000020135S1200000000000000
+  [red][T]0[D]S1200000000000004S0121300020135S1200000000000000
+  [green][T]0[D]S1200000000000004S013330020135S1200000000000000
+  [yellow][T]0[D]S1200000000000004S130330020135S1200000000000000
+  [blue][T]0[D]S1200000000000004S00120100020135S1200000000000000
+  [ok][T]0[D]S1200000000000004S10002100020135S1200000000000000
+  [up][T]0[D]S1200000000000004S12010000020135S1200000000000000
+  [down][T]0[D]S1200000000000004S000010000020135S1200000000000000
+  [left][T]0[D]S1200000000000004S00001300020135S1200000000000000
+  [right][T]0[D]S1200000000000004S1201300020135S1200000000000000
+  [vol+][T]0[D]S1200000000000004S1213000020135S1200000000000000
+  [vol-][T]0[D]S1200000000000004S00013000020135S1200000000000000
+  [mute][T]0[D]S1200000000000004S1003300020135S1200000000000000
+  [help][T]0[D]S1200000000000004S1300003020135S1200000000000000
diff -Naur remotes/dream.rem.orig remotes/dream.rem
--- remotes/dream.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/dream.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,43 @@
+[REMOTE]
+  [NAME]dream
+
+[TIMING]
+
+
+[COMMANDS]
+  [1][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1464 216 752 216 0 8 744 208 760 208 760 216 752 216 888 216 752 216 760 208 0 312 296 184 784 184 0 8 640 184 784 184 0 8 776 176 1328 208 1312 200 904 200 0 8 80 200 0 48 1472 264 704 232 1688 264 1792 264 704 264 704 264 840 264 704 264 704 240 
+  [2][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 64 216 0 48 1464 208 760 208 0 8 616 208 760 208 760 208 760 208 1032 208 760 216 752 216 0 312 296 208 760 208 0 8 616 208 760 208 0 8 744 216 1296 216 1296 216 888 216 0 8 64 216 0 48 1464 240 728 216 1568 216 1840 216 752 216 752 216 1024 216 752 192 776 192 
+  [3][RAW]91[FREQ]38[D]216 752 216 0 8 608 208 760 216 0 8 744 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 480 208 760 232 736 208 760 208 1168 208 760 208 760 208 0 312 296 232 736 232 0 8 592 232 736 264 0 8 696 256 1256 256 1256 256 848 264 0 8 16 256 0 48 1416 264 712 256 1392 256 1800 256 712 256 712 256 1120 256 712 256 712 264 
+  [4][RAW]91[FREQ]38[D]216 752 216 0 8 608 216 752 216 0 8 744 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 344 208 760 208 760 208 760 208 1304 208 760 208 760 208 0 312 296 232 736 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 200 904 200 0 8 80 200 0 48 1480 200 768 184 1328 184 1872 200 768 184 784 184 1328 184 784 184 784 184 
+  [5][RAW]91[FREQ]38[D]208 760 216 0 8 608 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1464 216 760 208 0 8 200 216 752 216 760 208 760 208 1440 208 760 208 760 208 0 312 296 208 760 216 0 8 608 208 760 208 0 8 752 208 1304 208 1304 208 896 240 0 8 40 208 0 48 1464 240 736 232 1144 232 1824 232 736 232 736 232 1416 232 736 232 736 240 
+  [6][RAW]91[FREQ]38[D]208 760 216 0 8 608 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 232 0 48 1440 216 760 208 0 8 64 240 728 216 760 232 736 208 1576 232 736 232 736 232 0 312 272 216 752 216 0 8 608 216 752 216 0 8 744 208 1304 208 1304 208 896 216 0 8 64 208 0 48 1464 216 752 216 1024 216 1848 208 760 208 760 208 1576 208 760 208 760 208 
+  [7][RAW]89[FREQ]38[D]208 760 216 0 8 608 208 760 216 0 8 744 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1464 216 760 208 1984 208 760 208 760 208 760 208 1712 208 760 208 760 208 0 312 296 216 752 216 0 8 608 208 760 240 0 8 720 208 1304 240 1272 208 896 240 0 8 40 240 0 48 1432 192 784 184 920 184 1872 184 784 184 784 184 1736 184 784 184 784 184 
+  [8][RAW]89[FREQ]38[D]184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 200 0 48 1480 200 768 200 1856 200 768 200 768 200 768 200 1856 200 768 200 768 200 0 312 312 232 736 232 0 8 592 224 744 232 0 8 728 224 1288 224 1288 224 880 224 0 8 56 224 0 48 1448 232 744 224 744 224 1832 224 744 224 744 224 1832 224 744 224 744 224 
+  [9][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1464 216 752 216 1704 216 752 216 752 216 752 216 1976 216 752 216 760 208 0 312 296 256 712 256 0 8 568 256 712 256 0 8 704 256 1176 336 1200 312 848 256 0 8 16 264 0 48 1416 264 704 264 0 8 696 264 1792 264 704 264 704 264 1928 264 704 264 704 264 
+  [0][RAW]89[FREQ]38[D]216 760 208 0 8 608 216 752 216 0 8 744 208 1304 208 1304 208 896 216 0 8 64 208 0 48 1472 208 760 208 760 208 760 208 760 208 760 208 760 216 752 216 752 216 0 312 296 240 728 240 0 8 584 232 736 232 0 8 728 232 1272 240 1280 232 872 232 0 8 40 240 0 48 1440 240 728 240 1816 240 1816 240 728 240 728 240 728 240 728 240 728 240 
+  [power][RAW]93[FREQ]38[D]232 736 232 0 8 592 232 736 232 0 8 728 232 1280 232 1280 232 872 232 0 8 48 232 0 48 1448 232 736 232 872 232 736 232 736 232 736 232 0 8 728 232 736 232 736 232 0 312 280 232 736 232 0 8 592 232 736 232 0 8 728 232 1280 232 1280 232 872 232 0 8 48 232 0 48 1448 232 736 232 1960 232 1824 232 736 232 736 232 0 8 728 224 744 232 736 232 
+  [num_left][RAW]91[FREQ]38[D]208 768 200 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 72 208 760 208 760 208 1440 208 896 208 760 208 760 208 0 312 304 208 760 208 0 8 616 208 760 208 0 8 752 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1496 184 784 184 1056 184 1872 184 784 184 1464 184 920 184 784 184 784 184 
+  [num_right][RAW]91[FREQ]38[D]232 744 224 0 8 592 232 736 232 0 8 728 232 1280 232 1280 208 896 232 0 8 48 208 0 48 1472 208 760 208 0 8 208 208 760 208 760 208 1440 208 760 208 760 208 760 208 0 312 304 232 736 232 0 8 592 232 736 232 0 8 728 232 1280 232 1280 232 872 232 0 8 48 232 0 48 1448 232 736 232 1144 232 1824 232 736 232 1416 232 736 256 712 232 736 264 
+  [mute][RAW]89[FREQ]38[D]208 768 200 0 8 624 200 768 200 0 8 760 200 1304 208 1312 200 904 200 0 8 80 200 0 48 1472 208 760 208 1848 208 760 208 760 208 1312 200 1312 200 768 200 768 200 0 312 304 208 768 200 0 8 616 208 768 200 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 760 208 1848 208 760 208 1304 208 1304 208 760 208 760 208 
+  [exit][RAW]89[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 200 1312 200 904 208 0 8 72 200 0 48 1472 208 768 200 1992 200 768 200 768 200 1448 200 1040 200 768 200 768 200 0 312 312 200 768 208 0 8 616 200 768 200 0 8 760 200 1312 200 1312 200 904 200 0 8 80 200 0 48 1472 208 760 208 904 200 1848 208 768 200 1448 200 1040 200 768 200 768 200 
+  [vol+][RAW]95[FREQ]38[D]200 768 208 0 8 616 200 768 208 0 8 752 200 1312 200 1312 200 904 200 0 8 80 200 0 48 1472 208 760 208 1576 208 768 200 768 200 768 200 0 8 80 200 768 200 768 208 0 312 304 208 760 232 0 8 592 200 768 208 0 8 752 224 1288 224 1288 224 880 224 0 8 56 200 0 48 1472 232 736 232 0 8 592 208 1848 208 760 208 760 208 0 8 72 208 760 208 760 208 
+  [vol-][RAW]95[FREQ]38[D]208 768 200 0 8 616 208 768 200 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 1440 208 760 208 760 208 760 208 0 8 208 208 760 208 768 200 0 312 304 240 736 232 0 8 584 232 736 232 0 8 728 232 1280 232 1280 232 872 232 0 8 48 232 0 48 1448 232 736 256 0 8 432 232 1824 232 736 256 712 256 0 8 160 232 736 256 712 264 
+   [bouq+][RAW]95[FREQ]38[D]184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1488 184 792 176 1200 184 784 184 784 184 784 184 0 8 504 184 784 184 784 184 0 312 328 184 784 184 0 8 640 176 792 184 0 8 776 176 1336 176 1336 176 928 176 0 8 104 176 0 48 1496 208 768 176 0 8 232 208 1848 184 784 184 784 232 0 8 456 264 704 264 704 264 
+  [bouq-][RAW]95[FREQ]38[D]176 792 184 0 8 640 176 792 184 0 8 776 176 1336 176 1336 176 928 176 0 8 104 176 0 48 1496 184 784 184 1064 176 792 176 792 176 792 176 0 8 648 176 792 176 792 176 0 312 328 208 768 200 0 8 624 192 776 176 0 8 776 200 1320 192 1312 200 904 200 0 8 80 200 0 48 1480 200 768 200 0 8 80 200 1856 200 768 200 768 200 0 8 624 200 768 200 768 200 
+  [info][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 232 1280 208 1304 208 896 208 0 8 72 232 0 48 1448 200 768 232 1552 208 760 208 760 208 1032 208 1848 208 760 208 760 208 0 312 304 208 760 208 0 8 616 208 760 208 0 8 752 208 1304 184 1328 184 920 184 0 8 96 184 0 48 1488 184 792 184 0 8 632 184 1872 184 784 184 1056 184 1880 176 792 176 792 184 
+  [menu][RAW]91[FREQ]38[D]208 768 200 0 8 616 208 760 208 0 8 752 208 1304 232 1280 232 872 240 0 8 40 232 0 48 1448 232 736 232 0 8 592 232 736 232 736 232 1008 232 736 232 736 240 728 240 0 312 272 184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1496 208 760 208 1576 208 1848 208 760 208 1032 208 760 208 760 208 760 208 
+  [audio][RAW]89[FREQ]38[D]184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 208 896 208 0 8 72 208 0 48 1472 208 760 208 1848 208 760 208 760 208 1032 208 1576 208 760 208 760 208 0 312 304 184 784 184 0 8 640 176 792 184 0 8 776 176 1336 176 1336 176 928 176 0 8 104 176 0 48 1496 184 784 184 792 176 1880 176 792 176 1064 176 1608 176 792 176 792 176 
+  [video][RAW]91[FREQ]38[D]208 768 200 0 8 616 208 768 200 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 1712 208 760 208 760 208 1032 208 1712 208 760 208 760 208 0 312 304 208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 752 208 1848 208 760 232 1008 232 1688 232 736 232 736 232 
+  [up][RAW]91[FREQ]38[D]184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 208 0 8 72 184 0 48 1488 192 784 184 0 8 504 176 792 184 784 184 1056 208 896 208 760 208 760 208 0 312 304 176 792 184 0 8 640 176 792 176 0 8 784 176 1336 176 1336 176 928 176 0 8 104 176 0 48 1496 184 784 184 1464 184 1872 184 784 184 1056 184 928 176 792 176 792 184 
+  [down][RAW]91[FREQ]38[D]208 768 208 0 8 616 200 768 200 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 344 208 760 208 760 208 1032 208 1032 208 760 208 760 240 0 312 272 184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 208 0 8 72 232 0 48 1448 200 768 232 1280 208 1848 200 768 208 1032 208 1032 232 736 208 760 208 
+  [right][RAW]91[FREQ]38[D]200 776 200 0 8 616 200 768 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 0 8 72 208 760 208 760 208 1032 208 1304 208 760 208 760 208 0 312 304 184 784 184 0 8 640 184 784 184 0 8 776 176 1336 176 1336 176 928 184 0 8 96 176 0 48 1496 216 752 240 1000 208 1848 208 768 200 1040 224 1288 200 768 208 760 208 
+  [left][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 232 0 8 48 208 0 48 1472 232 736 232 0 8 184 232 736 232 736 232 1008 232 1144 232 736 208 760 240 0 312 272 184 784 208 0 8 616 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1488 208 768 200 1176 224 1832 200 768 200 1040 200 1176 200 768 200 768 208 
+  [ok][RAW]89[FREQ]38[D]184 792 176 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1496 184 784 184 2008 184 784 184 784 184 1056 184 1464 184 784 184 784 184 0 312 328 264 704 264 0 8 560 256 712 256 0 8 704 256 1208 304 1208 304 848 256 0 8 24 256 0 48 1416 264 712 256 848 256 1800 256 712 256 984 256 1392 256 712 256 712 256 
+  [red][RAW]91[FREQ]38[D]184 784 184 0 8 640 184 784 184 0 8 776 184 1328 184 1328 184 920 184 0 8 96 184 0 48 1496 184 784 208 0 8 344 184 784 184 784 184 1328 208 760 208 760 208 760 208 0 312 304 184 784 184 0 8 640 176 792 184 0 8 776 176 1336 176 1336 176 928 176 0 8 104 176 0 48 1496 184 792 176 1336 176 1872 184 792 176 1336 176 792 176 792 176 792 184 
+  [green][RAW]91[FREQ]38[D]208 768 208 0 8 616 208 760 208 0 8 752 200 1312 200 1312 200 904 208 0 8 72 200 0 48 1472 208 760 208 0 8 208 208 760 208 760 208 1304 208 896 208 768 200 768 208 0 312 296 208 768 200 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 232 736 232 1144 232 1824 232 736 232 1280 232 872 232 736 240 728 240 
+  [yellow][RAW]91[FREQ]38[D]208 768 208 0 8 616 200 768 200 0 8 752 208 1304 240 1272 208 896 208 0 8 72 208 0 48 1472 208 760 232 0 8 48 232 736 232 736 232 1280 208 1032 232 736 208 760 240 0 312 272 208 760 208 0 8 616 208 760 232 0 8 728 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 232 736 208 1032 232 1824 232 736 232 1280 232 1008 232 736 232 736 232 
+  [blue][RAW]89[FREQ]38[D]208 760 208 0 8 616 232 736 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1464 208 760 208 1984 208 760 208 768 200 1312 200 1176 208 760 208 760 208 0 312 296 216 760 208 0 8 608 208 768 232 0 8 720 240 1272 208 1304 240 864 240 0 8 40 240 0 48 1440 232 736 208 896 240 1816 232 736 240 1272 240 1136 240 728 240 728 240 
+  [tv][RAW]95[FREQ]38[D]200 768 232 0 8 592 200 768 232 0 8 728 200 1312 224 1288 224 880 200 0 8 80 200 0 48 1472 208 768 200 1312 200 768 224 744 224 744 200 0 8 352 200 768 200 768 200 0 312 304 208 768 200 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 232 736 208 0 8 344 232 1824 232 736 232 736 232 0 8 320 232 736 232 736 232 
+  [radio][RAW]89[FREQ]38[D]208 768 208 0 8 616 200 768 232 0 8 720 208 1304 240 1272 240 864 208 0 8 72 240 0 48 1440 232 736 240 1816 232 736 240 728 240 1408 240 1136 240 728 240 728 240 0 312 272 208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 208 0 8 72 208 0 48 1472 208 760 208 760 208 1848 208 760 208 1440 208 1168 208 760 208 760 208 
+  [text][RAW]91[FREQ]38[D]208 760 208 0 8 616 208 760 208 0 8 752 208 1304 208 1304 208 896 232 0 8 48 208 0 48 1464 208 768 200 1720 200 768 200 768 208 1304 200 1448 224 744 200 768 208 0 312 304 208 760 232 0 8 592 208 760 208 0 8 744 240 1272 240 1272 240 864 240 0 8 40 240 0 48 1440 208 760 208 0 8 752 232 1824 232 736 232 1280 232 1416 232 736 232 736 232 
+  [help][RAW]91[FREQ]38[D]208 768 208 0 8 608 208 760 240 0 8 720 232 1280 232 1280 240 864 240 0 8 40 232 0 48 1448 232 736 232 1688 208 760 232 736 232 1416 232 1280 232 736 232 736 240 0 312 272 232 736 208 0 8 616 208 760 232 0 8 728 200 1312 200 1312 200 904 200 0 8 80 200 0 48 1472 208 768 224 0 8 728 208 1848 232 736 232 1416 232 1280 208 760 208 768 200 
diff -Naur remotes/irtrans.rem.orig remotes/irtrans.rem
--- remotes/irtrans.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/irtrans.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,40 @@
+[REMOTE]
+  [NAME]irtrans
+
+[TIMING]
+  [0][N]0[RC]2[RP]82[FREQ]36[RC5]
+
+[COMMANDS]
+  [power][T]0[D]11000111001100
+  [mute][T]0[D]11000111001101
+  [1][T]0[D]11000111000001
+  [2][T]0[D]11000111000010
+  [3][T]0[D]11000111000011
+  [4][T]0[D]11000111000100
+  [5][T]0[D]11000111000101
+  [6][T]0[D]11000111000110
+  [7][T]0[D]11000111000111
+  [8][T]0[D]11000111001000
+  [9][T]0[D]11000111001001
+  [0][T]0[D]11000111000000
+  [list][T]0[D]11000111100011
+  [speaker][T]0[D]11000111100100
+  [Up][T]0[D]11000111100000
+  [Down][T]0[D]11000111100001
+  [+][T]0[D]11000111010000
+  [-][T]0[D]11000111010001
+  [OK][T]0[D]11000111111011
+  [TV][T]0[D]11000111111111
+  [Radio][T]0[D]11000111110101
+  [Menu][T]0[D]11000111100101
+  [Text][T]0[D]11000111111100
+  [Help][T]0[D]11000111001110
+  [Select][T]0[D]11000111011110
+  [i][T]0[D]11000111111000
+  [AV][T]0[D]11000111101010
+  [Exit][T]0[D]11000111101001
+  [Timer][T]0[D]11000111101011
+  [Red][T]0[D]11000111110111
+  [Green][T]0[D]11000111110110
+  [Yellow][T]0[D]11000111110010
+  [Blue][T]0[D]11000111110100
diff -Naur remotes/mce-keyboard.rem.orig remotes/mce-keyboard.rem
--- remotes/mce-keyboard.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/mce-keyboard.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,86 @@
+[REMOTE]
+  [NAME]mce-keyboard
+  [RCV-LEN]9
+
+[TIMING]
+  [0][N]5[1]2784 280[2]328 280[3]320 600[4]592 320[5]600 624[RC]1[RP]0[FREQ]39[SB]
+
+[COMMANDS]
+  [00][T]0[D]K0000000000000000
+
+  [3A][T]0[D]K0111001000000000
+  [29][T]0[D]K0110101000000000
+  [02][T]0[D]K0011110000000000
+  [03][T]0[D]K0011111000000000
+  [04][T]0[D]K0100000000000000
+  [05][T]0[D]K0100001000000000
+  [06][T]0[D]K0100010000000000
+  [07][T]0[D]K0100011000000000
+  [08][T]0[D]K0100100000000000
+  [09][T]1[D]K0100101000000000
+  [0A][T]0[D]K0100110000000000
+  [0B][T]0[D]K0100111000000000
+  [0C][T]0[D]K0101101000000000
+  [0D][T]0[D]K0101110000000000
+  [0E][T]0[D]K0101010000000000
+  [0F][T]0[D]K0101011000000000
+  [10][T]0[D]K0010100000000000
+  [11][T]0[D]K0011010000000000
+  [12][T]0[D]K0001000000000000
+  [13][T]0[D]K0010101000000000
+  [14][T]0[D]K0010111000000000
+  [15][T]0[D]K0011100000000000
+  [16][T]0[D]K0011000000000000
+  [17][T]0[D]K0001100000000000
+  [18][T]0[D]K0010010000000000
+  [19][T]0[D]K0010011000000000
+  [1A][T]0[D]K0101111000000000
+  [1B][T]0[D]K0110000000000000
+  [1C][T]0[D]K0101000000000000
+  [1E][T]0[D]K0000100000000000
+  [1F][T]0[D]K0010110000000000
+  [20][T]0[D]K0000111000000000
+  [21][T]0[D]K0001001000000000
+  [22][T]0[D]K0001010000000000
+  [23][T]0[D]K0001011000000000
+  [24][T]0[D]K0001101000000000
+  [25][T]0[D]K0001110000000000
+  [26][T]0[D]K0001111000000000
+  [27][T]0[D]K0110011000000000
+  [28][T]0[D]K0110100000000000
+  [2B][T]0[D]K0110010000000000
+  [56][T]0[D]K1100100000000000
+  [2C][T]0[D]K0011101000000000
+  [2D][T]0[D]K0011011000000000
+  [2E][T]0[D]K0000110000000000
+  [2F][T]0[D]K0011001000000000
+  [30][T]0[D]K0000101000000000
+  [31][T]0[D]K0010001000000000
+  [32][T]0[D]K0010000000000000
+  [33][T]0[D]K0110110000000000
+  [34][T]0[D]K0110111000000000
+  [35][T]0[D]K0111000000000000
+  [39][T]0[D]K0101100000000000
+  [01][T]0[D]K0101001000000000
+  [3B][T]0[D]K0111010000000000
+  [3C][T]0[D]K0111011000000000
+  [3D][T]0[D]K0111100000000000
+  [3E][T]0[D]K0111101000000000
+  [3F][T]0[D]K0111110000000000
+  [40][T]0[D]K0111111000000000
+  [41][T]0[D]K1000000000000000
+  [42][T]0[D]K1000001000000000
+  [43][T]0[D]K1000010000000000
+  [44][T]0[D]K1000011000000000
+  [57][T]0[D]K1000100000000000
+  [58][T]0[D]K1000101000000000
+  [47][T]0[D]K1001010000000000
+  [4F][T]0[D]K1001101000000000
+  [4B][T]0[D]K1010000000000000
+  [50][T]0[D]K1010001000000000
+  [4D][T]0[D]K1001111000000000
+  [48][T]0[D]K1010010000000000
+  [51][T]0[D]K1001110000000000
+  [49][T]0[D]K1001011000000000
+  [52][T]0[D]K1001001000000000
+  [53][T]0[D]K1001100000000000
diff -Naur remotes/mce-mouse.rem.orig remotes/mce-mouse.rem
--- remotes/mce-mouse.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/mce-mouse.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,10 @@
+[REMOTE]
+  [NAME]mce-mouse
+  [RCV-LEN]1
+
+[TIMING]
+  [0][N]5[1]2784 280[2]328 280[3]320 600[4]592 320[5]600 624[RC]1[RP]0[FREQ]39[SB]
+
+[COMMANDS]
+  [0][T]0[D]M0000000000000000000
+
diff -Naur remotes/mediacenter.rem.orig remotes/mediacenter.rem
--- remotes/mediacenter.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/mediacenter.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,56 @@
+[REMOTE]
+  [NAME]mediacenter
+
+[TIMING]
+  [0][N]0[RC]1[RP]0[FREQ]36[SB][RC6]
+
+[COMMANDS]
+  [1][T]0[D]S11101010000000000011110000010000000001
+  [2][T]0[D]S11101010000000000011110000010000000010
+  [3][T]0[D]S11101010000000000011110000010000000011
+  [4][T]0[D]S11101010000000000011110000010000000100
+  [5][T]0[D]S11101010000000000011110000010000000101
+  [6][T]0[D]S11101010000000000011110000010000000110
+  [7][T]0[D]S11101010000000000011110000010000000111
+  [8][T]0[D]S11101010000000000011110000010000001000
+  [9][T]0[D]S11101010000000000011110000010000001001
+  [0][T]0[D]S11101010000000000011110000010000000000
+  [ok][T]0[D]S11101010000000000011110000010000100010
+  [left][T]0[D]S11101010000000000011110000010000100000
+  [right][T]0[D]S11101010000000000011110000010000100001
+  [down][T]0[D]S11101010000000000011110000010000011111
+  [up][T]0[D]S11101010000000000011110000010000011110
+  [play][T]0[D]S11101010000000000011110000010000010110
+  [power][T]0[D]S11101010000000000011110000010000001100
+  [Stop][T]0[D]S11101010000000000011110000010000011001
+  [Rec][T]0[D]S11101010000000000011110000010000010111
+  [Rew][T]0[D]S11101010000000000011110000010000010101
+  [Fwd][T]0[D]S11101010000000000011110000010000010100
+  [Pause][T]0[D]S11101010000000000011110000010000011000
+  [Next][T]0[D]S11101010000000000011110000010000011010
+  [Prev][T]0[D]S11101010000000000011110000010000011011
+  [eHome][T]0[D]S11101010000000000011110000010000001101
+  [Back][T]0[D]S11101010000000000011110000010000100011
+  [EPG][T]0[D]S11101010000000000011110000010000100110
+  [Info][T]0[D]S11101010000000000011110000010000001111
+  [LiveTV][T]0[D]S11101010000000000011110000010000100101
+  [Video][T]0[D]S11101010000000000011110000010001001010
+  [Music][T]0[D]S11101010000000000011110000010001000111
+  [TV][T]0[D]S11101010000000000011110000010001000110
+  [Pictures][T]0[D]S11101010000000000011110000010001001001
+  [Vol+][T]0[D]S11101010000000000011110000010000010000
+  [Vol-][T]0[D]S11101010000000000011110000010000010001
+  [Mute][T]0[D]S11101010000000000011110000010000001110
+  [Ch+][T]0[D]S11101010000000000011110000010000010010
+  [Ch-][T]0[D]S11101010000000000011110000010000010011
+  [Enter][T]0[D]S11101010000000000011110000010000001011
+  [Clear][T]0[D]S11101010000000000011110000010000001010
+  [DVDMenu][T]0[D]S11101010000000000011110000010000100100
+  [Rectv][T]0[D]S11101010000000000011110000010001001000
+  [Messenger][T][0][D]S11101010000000000011110000010001100100
+  [Radio][T][0][D]S11101010000000000011110000010001010000
+  [Teletext][T][0][D]S11101010000000000011110000010001011010
+  [Red][T][0][D]S11101010000000000011110000010001011011
+  [Green][T][0][D]S11101010000000000011110000010001011100
+  [Yellow][T][0][D]S11101010000000000011110000010001011101
+  [Blue][T][0][D]S11101010000000000011110000010001011110
diff -Naur remotes/pace.rem.orig remotes/pace.rem
--- remotes/pace.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/pace.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,46 @@
+[REMOTE]
+  [NAME]pace
+
+[TIMING]
+  [0][N]1[1]0 0[RC]2[RP]100[IRDA-RAW]
+  [1][N]1[1]0 0[RC]2[RP]100[IRDA]
+
+[COMMANDS]
+  [0][T]0[D]0111110001011111111100010011110111111111010000011101011100110000011111
+  [1][T]0[D]0111110001011111111100101101110111111111010000111100001000110110000011
+  [2][T]0[D]0111110001011111111101000011110111111111010000111101101001110100101111
+  [3][T]0[D]0111110001011111111101001011110111111111010000101100110001110111101011
+  [4][T]0[D]0111110001011111111101101101110111111111010000011101010011110111011011
+  [5][T]0[D]0111110001011111111100100011110111111111010000011100000101110111000011
+  [6][T]0[D]0111110001011111111100101011110111111111010000001101011101110100000111
+  [7][T]0[D]0111110001011111111100011101110111111111010000111101010001110001011111
+  [8][T]0[D]0111110001011111111101100011110111111111010000111101011110110110011011
+  [9][T]0[D]0111110001011111111101101011110111111111010000101100000110110101011111
+  [Text][T]0[D]0111110001011111111101011011110111111111010000101101011111110010000011
+  [TEXTOFF][T]0[D]0111110001011111111101101111110111111111010000001100110110110011010011
+  [RED][T]0[D]0111110001011111111101111111110111111011000001101101010001110100100111
+  [GREEN][T]0[D]0111110001011111111101111111110111111101000001001101111011110010110011
+  [YELLOW][T]0[D]0111110001001111111101111111110111111111000001011101101100110101101011
+  [BLUE][T]0[D]0111110001010111111101111111110111111111010001101100010010110100110011
+  [Menu][T]0[D]0111110001011111111101010111110111111111010000011100110111110111001011
+  [Back][T]0[D]0111110001011111111100001101110111111111010000111100111111110100110111
+  [Guide][T]0[D]0111110001011111111101100111110111111111010000011101101110110000010111
+  [Exit][T]0[D]0111110001011111111100010111110111111111010000111101101100110110010011
+  [Info][T]0[D]0111110001011111111101001101110111111111010000011101100100110101101111
+  [Fav][T]0[D]0111110001011111111100001011110111111111010000001101101010110110110011
+  [Audio][T]0[D]0111110001011111111101011111110111111111010000001101101111110100001111
+  [Subtitle][T]0[D]0111110001011111111100011011110111111111010000001100000100110011011011
+  [Ch+][T]0[D]0111110001011111111100101111110111111111010000101101101101110010001011
+  [Ch-][T]0[D]0111110001011111111100100111110111111111010000111100110101110001001111
+  [Prev][T]0[D]0111110001011111111100000011110111111111010000011100110010110101110111
+  [Next][T]0[D]0111110001011111111100011111110111111111010000101100110100110101010111
+  [Power][T]0[D]0111110001011111111101001111110111111111010000001100000001110001100111
+  [Vol-][T]0[D]0111110001011111111100000111110111111111010000111100000010110011111011
+  [Vol+][T]0[D]0111110001011111111100001111110111111111010000101101011010110000111111
+  [Mute][T]0[D]0111110001011111111101000111110111111111010000011101011001110010100011
+  [Up][T]0[D]0111110001011111111101111111110101111111000001001100010000110101100111
+  [Down][T]0[D]0111110001011111111101111111110110111111000001001101111000110000110111
+  [Right][T]0[D]0111110001011111111101111111110111101111000001111100000100110111101111
+  [Left][T]0[D]0111110001011111111101111111110111011111010001011100111110110110011011
+  [OK][T]0[D]0111110001011111111101111111110011111111010001001100110110110011010111
+  [0-][T]1[D]11111000111111110010011111111111100000111011100100001111
diff -Naur remotes/routing.sample.orig remotes/routing.sample
--- remotes/routing.sample.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/routing.sample	2008-07-18 18:47:12.000000000 +0000
@@ -0,0 +1,14 @@
+[ADDRESS]
+  [NAME]Wohnzimmer[ADDRESS]0
+  [NAME]Schlafzimmer[ADDRESS]1
+  [NAME]Arbeitszimmer[ADDRESS]2
+
+[SEND-ROUTING]
+  [REM]Sony[ADDRESS]0xff
+  [REM]Yamaha[ADDRESS]0xffff
+[END]
+
+[RECV-ROUTING]
+  [REM]Tv[ADDRESS]0xffff
+  [REM]R1[ADDRESS]0xffff
+[END]
\ No newline at end of file
diff -Naur remotes/sky-digital.rem.orig remotes/sky-digital.rem
--- remotes/sky-digital.rem.orig	1970-01-01 00:00:00.000000000 +0000
+++ remotes/sky-digital.rem	2008-07-18 18:46:47.000000000 +0000
@@ -0,0 +1,38 @@
+[REMOTE]
+  [NAME]sky-digital
+
+[TIMING]
+  [0][N]0[RC]1[NOTOG1][RP]81[FREQ]36[SB][RS][RC6]
+
+[COMMANDS]
+  [TV][T]0[D]S11100000000000000010000100
+  [Sky][T]0[D]S11100000000000000010000000
+  [Power][T]0[D]S11100000000000000000001100
+  [TVGuide][T]0[D]S11100000000000000011001100
+  [BoxOffice][T]0[D]S11100000000000000001111101
+  [Services][T]0[D]S11100000000000000001111110
+  [Interactive][T]0[D]S11100000000000000011110101
+  [Select][T]0[D]S11100000000000000001011100
+  [SelectLeft][T]0[D]S11100000000000000001011010
+  [SelectRight][T]0[D]S11100000000000000001011011
+  [SelectUp][T]0[D]S11100000000000000001011000
+  [SelectDown][T]0[D]S11100000000000000001011001
+  [Ch+][T]0[D]S11100000000000000000100000
+  [Ch-][T]0[D]S11100000000000000000100001
+  [Text][T]0[D]S11100000000000000000111100
+  [BackUp][T]0[D]S11100000000000000010000011
+  [Help][T]0[D]S11100000000000000010000001
+  [TxtRed][T]0[D]S11100000000000000001101101
+  [TxtGreen][T]0[D]S11100000000000000001101110
+  [TxtYellow][T]0[D]S11100000000000000001101111
+  [TxtBlue][T]0[D]S11100000000000000001110000
+  [1][T]0[D]S11100000000000000000000001
+  [2][T]0[D]S11100000000000000000000010
+  [3][T]0[D]S11100000000000000000000011
+  [4][T]0[D]S11100000000000000000000100
+  [5][T]0[D]S11100000000000000000000101
+  [6][T]0[D]S11100000000000000000000110
+  [7][T]0[D]S11100000000000000000000111
+  [8][T]0[D]S11100000000000000000001000
+  [9][T]0[D]S11100000000000000000001001
+  [0][T]0[D]S11100000000000000000000000

